guides/source/configuring.md

```diff
@@ -1520,6 +1521,17 @@ The default value depends on the `config.load_defaults` target version:
 | (original)            | `true`               |
 | 7.0                   | `false`              |
 
+#### `config.action_view.prepend_content_exfiltration_prevention`
+
+Determines whether or not the `form_tag` and `button_to` helpers will produce HTML tags prepended with additional matter designed to prevent exfiltration of form data. The default value is `false`.
```

Comment:
"additional matter" feels a little bit too vague to me... given it's a security-related setting, there's admittedly a good chance that anyone who really cares will just go find the source, but still: perhaps some variant of "browser-safe (but technically invalid) HTML that guarantees their contents cannot be captured by any preceding unclosed tags"?